### PR TITLE
Google+ Profile Image is displayed on login in with Google Sign in

### PR DESCRIPTION
**Pull request by er-anup**
Made Changes in 
1) notifications.py : Imported urllib2 and json library. Add Google profile image url in session db
2) volunteercard.html : Displayed the image of Google+ profile.

### CODE REVIEW

**Comment by RaghavPro** [comment]
No need to store the image url. What if user changes it? Get it every time using the id.
Also, replace your key with old key in default_settings and get it like `app.config['GOOGLE_KEY_WHATEVER']`

**Comment by er-anup** [comment]
I have written function in helpers.py
changed /cleansweep/plugins/volunteers/templates/Volunteers.html and /cleansweep/plugins/volunteers/templates/widgets/VolunteeCcard.html

File: cleansweep/helpers.py [hunk 1]
```diff
@@ -150,6 +151,13 @@ def starts_with_vowel(input_text):
     return False
 
 @app.context_processor
+def google_image_url():
+    def get_google_photo_url(google_id):
+        google_profile_image_url = json.loads(urllib2.urlopen( "https://www.googleapis.com/plus/v1/people/"+google_id+"?fields=image&key="+app.config['GOOGLE_API_KEY']).read())['image']['url']
+        return google_profile_image_url
+    return dict(get_google_photo_url = get_google_photo_url)
+
```
Comment (anandology) [inline cleansweep/helpers.py]: Please move the get_google_photo_url to the top-level and add it to the list of functions exposed in helpers function below. There is no need to write google_image_url and the decorator.

Comment (anandology) [inline cleansweep/helpers.py]: Also, could you please add a check if `GOOGLE_API_KEY` is available in app.config. If it is not available, return the default image.

**Comment by er-anup** [comment]
Removed Decorator Function from get_google_photo_url
Add get_google_photo_url in helpers function

**Comment by anandology** [comment]
squashed and merged the commits. see e095186 and 2329ca2.

**Pull request closed without merging**